### PR TITLE
Update LBandConfig to add additonal fields and default values.

### PR DESCRIPTION
**Pull request by axlan**

### CODE REVIEW

File: src/point_one/fusion_engine/messages/configuration.h [hunk 1]
```diff
@@ -1439,28 +1439,65 @@ struct P1_ALIGNAS(4) TroposphereConfig {
...
+  uint64_t pmp_unique_word = 0;
 
   /** Data rate of the provider (bps). */
   uint16_t pmp_data_rate_bps = 0;
 
-  uint8_t reserved[2] = {0};
-
-  /** Unique word of the provider. */
-  uint32_t pmp_unique_word = 0;
+  /** The initialization value for the descrambler vector */
```
Comment (adamshapiro0) [inline src/point_one/fusion_engine/messages/configuration.h:1499]: "descrambling vector."

File: src/point_one/fusion_engine/messages/configuration.h [hunk 2]
```diff
@@ -1439,28 +1439,65 @@ struct P1_ALIGNAS(4) TroposphereConfig {
...
+   */
+  bool use_custom_descrambler_init = false;
+
+  /**
+   * If `true`, only output data frames with the configured service ID.
+   * Otherwise, output all decoded frames.
+   */
+  bool filter_data_by_service_id = true;
+
+  /** Enable/disable the descrambler */
```
Comment (adamshapiro0) [inline src/point_one/fusion_engine/messages/configuration.h:1487]: Missing .

File: src/point_one/fusion_engine/messages/configuration.h [hunk 3]
```diff
@@ -1439,28 +1439,65 @@ struct P1_ALIGNAS(4) TroposphereConfig {
...
+  /**
+   * The center frequency of the L-band beam (Hz). If `NAN` use the system
+   * default value.
+   */
+  double center_frequency_hz = NAN;
 
   /**
    * The size of the signal acquisition search space (in Hz) around the center
-   * frequency.
+   * frequency. If `NAN` use the system default value.
```
Comment (adamshapiro0) [inline src/point_one/fusion_engine/messages/configuration.h:1450]: Missing comma: If nan,

Here and for center freq

File: src/point_one/fusion_engine/messages/configuration.h [hunk 4]
```diff
@@ -1439,28 +1439,65 @@ struct P1_ALIGNAS(4) TroposphereConfig {
...
+  uint64_t pmp_unique_word = 0;
 
   /** Data rate of the provider (bps). */
   uint16_t pmp_data_rate_bps = 0;
 
-  uint8_t reserved[2] = {0};
-
-  /** Unique word of the provider. */
-  uint32_t pmp_unique_word = 0;
+  /** The initialization value for the descrambling vector */
```
Comment (adamshapiro0) [inline src/point_one/fusion_engine/messages/configuration.h:1499]: Missing .

File: python/fusion_engine_client/messages/configuration.py [hunk 5]
```diff
@@ -666,28 +666,50 @@ class LBandConfig(NamedTuple):
         @brief Configuration of the L-band demodulator parameters.
         """
         ## The center frequency of the L-band beam (Hz).
-        center_frequency_hz: float
+        center_frequency_hz: float = math.nan
```
Comment (anathan) [inline python/fusion_engine_client/messages/configuration.py:669]: lets just set this to the west coast (which is also the east coast)

File: src/point_one/fusion_engine/messages/configuration.h [hunk 6]
```diff
@@ -1439,28 +1439,61 @@ struct P1_ALIGNAS(4) TroposphereConfig {
...
+   * frequency. If `NAN`, use the system default value.
    *
    * For example, a value of 6000 will search +/- 3 kHz around the center
    * frequency.
    */
   float search_window_hz = NAN;
 
+  /**
+   * If `true`, use the provider date rate in this configuration. Otherwise, use
+   * device's default value.
```
Comment (anathan) [inline src/point_one/fusion_engine/messages/configuration.h:1459]: this is a nit, but the comment should mention WHAT the default value is. (repeat for all of these)

File: src/point_one/fusion_engine/messages/configuration.h [hunk 7]
```diff
@@ -1439,28 +1439,61 @@ struct P1_ALIGNAS(4) TroposphereConfig {
...
+  bool filter_data_by_service_id = true;
+
+  /** Enable/disable the descrambler. */
+  bool use_descrambler = true;
+
+  uint8_t reserved[1] = {0};
+
   /** Service ID of the provider. */
-  uint32_t pmp_service_id = 0;
+  uint16_t pmp_service_id = 0x5555;
```
Comment (anathan) [inline src/point_one/fusion_engine/messages/configuration.h:1487]: should we make all of these be the defaults for our service? That seems to make sense to me.

**Review by adamshapiro0 — APPROVED** [review]
LGTM

**Pull request merged**